test/ex_machina_test.exs

```diff
@@ -98,4 +98,23 @@ defmodule ExMachinaTest do
     }
     assert records == [expected_record, expected_record, expected_record]
   end
+
+  test "create/2 raises helpful error denoting use of insert/2" do
+    assert_raise RuntimeError, ~r/create\/2 has been removed/, fn ->
+      Factory.create(:user, admin: true)
+    end
+  end
+
+  test "create_pair/2 raises helpful error denoting use of insert_pair/2" do
+    assert_raise RuntimeError, ~r/create_pair\/2 has been removed/, fn ->
+      Factory.create_pair(:user, admin: true)
+    end
+  end
+
+  test "create_list/3 raises helpful error denoting use of insert_list/3" do
+    assert_raise RuntimeError, ~r/create_list\/3 has been removed/, fn ->
+      Factory.create_list(3, :user, admin: true)
+    end
+  end
```

Comment:
Since these errors are so similar I think they could all go in on test like `raises helpful errors when using the old create functions` and then it has all the assert_raises in that one test. What do you think?